src/main/resources/edu/cornell/kfs/pdp/businessobject/datadictionary/ACHPayee.xml

```diff
@@ -58,8 +58,43 @@
             </list>
           </property>
         </bean>
+        
+        <!-- Add "principalName" to the ACHPayee lookup results. -->
+        <bean parent="FieldOverrideForListElementInsert">
+          <property name="propertyName" value="formAttributeDefinitions" />
+          <property name="propertyNameForElementCompare" value="name" />
+          <property name="element">
+            <ref bean="DisbursementPayee-payeeIdNumber"/>

```

Comment:
I believe the "formAttributeDefinitions" comparison should check against the `ACHPayee-entityId` bean instead. Likewise, the "displayAttributeDefinitions" comparison should check against the `DisbursementPayee-payeeIdNumber` bean instead. Sorry if that suggestion from my previous review wasn't clear.

Note that if the comparison element is not found in the list, I think KFS will stick the insertion beans at the end of the list instead. That explains why KFS can still launch even though these comparison elements cannot be found.

Reply:
Good catch! Updating